pkg/election/election_test.go

```diff
@@ -73,10 +75,15 @@ func (t *testElectionSuite) SetUpTest(c *C) {
 	case <-time.After(10 * time.Second):
 		c.Fatal("start embed etcd timeout")
 	}
+
+	// some notify leader information is not handled, just reduce the block time and ignore them
+	t.originNotifyBlockTime = NotifyBlockTime
+	NotifyBlockTime = 100 * time.Millisecond
```

Comment:
How about passing the block time to `NewElection` as an argument?

Reply:
update in 106b035